src/printer.js

```diff
@@ -510,6 +510,12 @@ function genericPrintNoParens(path, options, print) {
       if (hasContent) {
         parts.push(indent(options.tabWidth, concat([ hardline, naked ])));
       }
+      else if(n.comments) {
+        parts.push(
+          indent(options.tabWidth,
```

Comment:
yeah, because we haven't automatically hooked it up I just code whatever and know it'll be formatted at some point. I'll change this though!

We're probably stable enough to try to enforce PRs to be formatted. I should at least turn on Emacs integration again. We weren't stable enough before but we probably are now.